From: Benjamin Drung <benjamin.drung@canonical.com>
Date: Wed, 6 Aug 2025 15:13:06 +0200
Subject: test: drop requiring iscsiuio.socket

The iscsiuio socket/service is only needed on specific hardware. It is
not needed for the ISCSI and ISCSI-MULTI test cases. Without this check
the tests will succeed on Debian and Ubuntu.

Forwarded: https://github.com/dracut-ng/dracut-ng/pull/1528
---
 .github/workflows/integration.yml | 2 ++
 test/TEST-70-ISCSI/test.sh        | 4 ----
 test/TEST-71-ISCSI-MULTI/test.sh  | 4 ----
 3 files changed, 2 insertions(+), 8 deletions(-)

diff --git a/.github/workflows/integration.yml b/.github/workflows/integration.yml
index 88c70bb..1c90460 100644
--- a/.github/workflows/integration.yml
+++ b/.github/workflows/integration.yml
@@ -197,7 +197,9 @@ jobs:
             matrix:
                 container:
                     - arch:latest
+                    - debian:latest
                     - opensuse:latest
+                    - ubuntu:rolling
                 test:
                     - "70"
                     - "71"
diff --git a/test/TEST-70-ISCSI/test.sh b/test/TEST-70-ISCSI/test.sh
index 3eb6d70..83b1c65 100755
--- a/test/TEST-70-ISCSI/test.sh
+++ b/test/TEST-70-ISCSI/test.sh
@@ -122,10 +122,6 @@ test_check() {
         echo "Need tgtd and tgtadm from scsi-target-utils"
         return 1
     fi
-    if ! [ -f /lib/systemd/system/iscsiuio.socket ]; then
-        echo "Need iscsiuio.socket to run this test"
-        return 1
-    fi
 }
 
 test_setup() {
diff --git a/test/TEST-71-ISCSI-MULTI/test.sh b/test/TEST-71-ISCSI-MULTI/test.sh
index f5c3e24..55eae0f 100755
--- a/test/TEST-71-ISCSI-MULTI/test.sh
+++ b/test/TEST-71-ISCSI-MULTI/test.sh
@@ -129,10 +129,6 @@ test_check() {
         echo "Need tgtd and tgtadm from scsi-target-utils"
         return 1
     fi
-    if ! [ -f /lib/systemd/system/iscsiuio.socket ]; then
-        echo "Need iscsiuio.socket to run this test"
-        return 1
-    fi
 }
 
 test_setup() {
